.github/workflows/CI.yml

```diff
@@ -15,26 +15,13 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - "1.3"  # Minimal version of the package
+          - "1.6"  # Minimal version of the package
           - "1"    # Latest Version
         os:
           - ubuntu-latest
-          - macOS-latest
-          - windows-latest
         arch:
           - x64
           - x86
-        exclude:
-          # Test 32-bit only on Linux
-          - os: macOS-latest
-            arch: x86
-          - os: windows-latest
-            arch: x86
-        include:
-          # Add a 1.6 job because that's what Invenia actually uses
-          - os: ubuntu-latest
-            version: 1.6
-            arch: x64
```

Comment:
I can't comment this far down in the file, but we can remove the Slack notification on failure section.